modules/geoip2/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+option(ENABLE_GEOIP2 "Enable geoip2 parser and template function" ON)
+
+if (NOT ENABLE_GEOIP2)
```

Comment:
If there is no `libmaxminddb` in the system, the `cmake` step going to fail by default.
I think it is more desirable to disable the module(s) if the user have not stated otherwise.
```CMake
find_package(LibMaxMinDB)
if (LIBMAXMINDB_FOUND)
  option(ENABLE_GEOIP2 "Enable geoip2 parser and template function" ON)
else()
  option(ENABLE_GEOIP2 "Enable geoip2 parser and template function" OFF)
endif()
```
will achieve that. But the user still could override this via
```bash
cmake -DENABLE_GEOIP2=[on/off]
```
The error message could still be written, because in this case the `ENABLE_GEOIP2` exists before the `option(ENABLE_GEOIP2...` line.